udata/core/dataset/rdf.py

```diff
@@ -338,6 +345,10 @@ def resource_from_rdf(graph_or_distrib, dataset=None):
     download_url = url_from_rdf(distrib, DCAT.downloadURL)
     access_url = url_from_rdf(distrib, DCAT.accessURL)
     url = safe_unicode(download_url or access_url)
+    # we shouldn't create resources without URLs
+    if not url:
```

Comment:
Do not create resources without urls.

Reply:
No warning will be shown on admin's page though?

Reply:
True, I'll look into it and see if it's easy to pop a warning.